### DISCUSSION TITLE
Not receiving data from Kamstrup meter anymore

### DISCUSSION THREAD

**Discussion by tlvestergaard** [UtilitechAS/amsreader-firmware#685] [Q&A] [answered]
Up until last week my Pow-K+ module has worked flawlessly. But 2023-11-27, 10.00 pm was the last time my Pow-K+ received data from the Kamstrup meter.
At that time I was on SW 2.2.16. I have since upgraded to 2.2.23 and 2.2.24. I have done a factory reset, I have unplugged the Pow module from the meter and inserted it again. But still, no data. 
I have contacted Nord Energi Net. They tell me all is okay at their end. They sent me the two encryptions keys again, which I have tried, but still, no data.
The webserver part of Pow-K+is responsive, and price info can be fetched from Entsoe. 
The LED on the PowK module is blinking red every 5-6 seconds. There is no blue blink every 10 seconds to indicate data.
I notice on the attached screenshot that the PowK has not detected any meter. That can't be good?

Any suggestions on what I can try next? 
Could the Pow-K+ module be broken, or should I turn my suspicion towards Nord Energi Net?

![amsleser1](https://github.com/UtilitechAS/amsreader-firmware/assets/7568204/2207fd5d-fdad-46f0-974b-f5a2fe8e6339)
![amsleser2](https://github.com/UtilitechAS/amsreader-firmware/assets/7568204/f67af4a5-d6cd-4b83-af73-10170b1ca372)

**Comment by ArnieO** [comment, marked as answer]
We've seen this before, happens from time to time with the Kamstrup Omnipower meters: They just stop delivering data on the HAN port. Grid company can't see it on their side. 

Your most probable solution is to do a reboot of the meter. 
Nordnet can do a remote reboot, so I would contact them and request that.

**Comment by tlvestergaard** [comment]
Hey ArnieO,
Thanks for your answer. It seems very plausible that that is what I am experiencing.
Nord Energi Net claim they cannot remotely reset the meter, but that they need to send a technician - at my expense. 
I am not sure I want, or should, for pay for that, so the issue remains unsolved for now, while I am considering what I will do.

**Reply by ArnieO** [reply]
That is strange, maybe they lack a tool that most other grid companies have. 
An alternative solution, providing your main breaker is on the "street side" of the meter is to do a "manual reset" by power cycling the meter:
Pull the main fuse for 30 seconds or so - then close it again. The disadvantage is of course that all mains-powered electronics will reset/reboot.

**Reply by tlvestergaard** [reply]
Issue resolved.

There is no breaker on the "street side" of the meter, so I can not do anything myself. But I have had a certified electrician to disconnect power to the meter. Waited a minute or so. Connected power again. 
Within a very short time there were blue flashes from Pow-K, and when my Wi-Fi was up again, it was clear to see on Pow-K's web interface that it was receiving data from the Kamstrup meter again.
This leads me to conclude, that the problem was the Kamstrup meter - just as you suggested. 
Thank you for your help, ArnieO!

**Reply by ArnieO** [reply]
Good to hear you got it sorted out!
The strange point here is that your grid company Nord Energi Net claimed they cannot remotely reset the meter. As far as we know, that can be done in the same tool they use for configuring the meter (Omnisoft Vision Air). We have seen several cases in Denmark where the grid company did such a remote reset. Maybe you were just unlucky with the person that handled your inquiry, that he/she was not aware of that possibility?

I have now sent an email to info@nordenergi.dk to try to get to the bottom of this - let us see what their answer is!
